### PR TITLE
js dependencies

### PR DESCRIPTION
**Pull request by wadie**
I still need to compile into a minified main.js and add Zepto script to header.html

will be merged with #241

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,7 +6,8 @@
   "scripts": {
     "http-server": "./node_modules/.bin/http-server",
     "sass": "sass --watch ./application/stylesheets/scss/custom.bootstrap.scss:./application/stylesheets/css/custom.bootstrap.css",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "conpile": ""
```
Comment (ghislaineguerin) [inline package.json]: @wadie is this a typo?

Comment (wadie) [reply package.json]: Fixed, it's not needed.

Comment (ghislaineguerin) [reply package.json]: 👍

**Comment by ghislaineguerin** [comment]
@wadie do we have gulp watch for watching both js and CSS? this would replace sass watch I assume

File: gulpfile.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    watch = require('gulp-watch');
+    terser = require('gulp-terser');
+
+gulp.task('js', function(){
+    return gulp.src('./js/*.js')
+        .pipe(gp_concat('main.js'))
+        .pipe(gulp.dest('./js/'))
+        .pipe(gp_rename('main.js'))
+        .pipe(terser())
+        .pipe(gulp.dest('./js/'));
```
Comment (ghislaineguerin) [inline gulpfile.js]: @wadie we are missing this for when there are CSS changes right?

**Pull request merged**